packages/utilities/src/m3/infer-languages.ts

```diff
@@ -58,12 +58,16 @@ export const inferLanguagesFromSerializationChunk = (chunk: SerializationChunk):
                 id(languageName, concept.name, propertyName)
             ).havingKey(property.property.key)
 
-            if (isBoolean(property.value)) {
-                feature.ofType(booleanDatatype)
-            } else if (isNumeric(property.value)) {
-                feature.ofType(integerDatatype)
+            if (property.value === null) {
+                feature.isOptional()
```

Comment:
Surprised at the naming convention here. Usually an `isSomething()` method returns whether the `Something` is true (or not).  But this is more of a a `setOptional(...)`

Reply:
It's part of the fluent-style API for the TypeScript LionCore M3. I like the style, but I also understand how it can be confusing.

Reply:
I understand, never found the fluent style very appealing.